### DISCUSSION TITLE
今天重启之后无法同步媒体库数据；无法同步CookieCloud；无法连接QB，回退版本依旧

### DISCUSSION THREAD

**Discussion by kingcard1131** [jxxghp/MoviePilot#1188] [问题咨询]
前端版本：v1.4.8，后端版本：v1.4.7
运行环境:群辉 Docker
1、无法同步媒体库数据
2、无法同步CookieCloud
3、无法连接QB

那位大佬知道是什么问题，之前一直好好的

日志：
moviepilot-moviepilot-1  | ERROR:   emby.py - Users 未获取到返回数据
moviepilot-moviepilot-1  | ERROR:   emby.py - Library/SelectableMediaFolders 未获取到返回数据
moviepilot-moviepilot-1  | INFO:    module.py - Moudle Loaded：EmbyModule
moviepilot-moviepilot-1  | INFO:    module.py - Moudle Loaded：FanartModule
moviepilot-moviepilot-1  | INFO:    module.py - Moudle Loaded：FileTransferModule
moviepilot-moviepilot-1  | INFO:    module.py - Moudle Loaded：FilterModule
moviepilot-moviepilot-1  | INFO:    module.py - Moudle Loaded：IndexerModule
moviepilot-moviepilot-1  | WARNING: connectionpool.py - Retrying (Retry(total=0, connect=0, read=1, redirect=None, status=None)) after connection broken by 'ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f47ef8dae50>, 'Connection to 192.168.31.143 timed out. (connect timeout=15)')': /
moviepilot-moviepilot-1  | WARNING: connectionpool.py - Retrying (Retry(total=0, connect=0, read=1, redirect=None, status=None)) after connection broken by 'ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f47ef8d9350>, 'Connection to 192.168.31.143 timed out. (connect timeout=15)')': /
moviepilot-moviepilot-1  | WARNING: connectionpool.py - Retrying (Retry(total=0, connect=0, read=1, redirect=None, status=None)) after connection broken by 'ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f47ef712c90>, 'Connection to 192.168.31.143 timed out. (connect timeout=15)')': /api/v2/auth/login
moviepilot-moviepilot-1  | WARNING: connectionpool.py - Retrying (Retry(total=0, connect=0, read=1, redirect=None, status=None)) after connection broken by 'ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f47ef719650>, 'Connection to 192.168.31.143 timed out. (connect timeout=15)')': /
moviepilot-moviepilot-1  | WARNING: connectionpool.py - Retrying (Retry(total=0, connect=0, read=1, redirect=None, status=None)) after connection broken by 'ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f47f10f5010>, 'Connection to 192.168.31.143 timed out. (connect timeout=15)')': /
moviepilot-moviepilot-1  | WARNING: connectionpool.py - Retrying (Retry(total=0, connect=0, read=1, redirect=None, status=None)) after connection broken by 'ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f47ef711d10>, 'Connection to 192.168.31.143 timed out. (connect timeout=15)')': /api/v2/auth/login
moviepilot-moviepilot-1  | ERROR:   qbittorrent.py - qbittorrent 连接出错：Failed to connect to qBittorrent. Unknown Error: ConnectTimeout(MaxRetryError("HTTPConnectionPool(host='192.168.31.143', port=9888): Max retries exceeded with url: /api/v2/auth/login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f47ef710b90>, 'Connection to 192.168.31.143 timed out. (connect timeout=15)'))"))

moviepilot-moviepilot-1  | ERROR:   emby.py - User/Views 未获取到返回数据
moviepilot-moviepilot-1  | INFO:    mediaserver.py - 【MediaServer】媒体库数据同步完成，同步数量：0

moviepilot-moviepilot-1  | ERROR:   qbittorrent.py - qbittorrent 连接出错：Failed to connect to qBittorrent. Unknown Error: ConnectTimeout(MaxRetryError("HTTPConnectionPool(host='192.168.31.143', port=9888): Max retries exceeded with url: /api/v2/auth/login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fa789728c10>, 'Connection to 192.168.31.143 timed out. (connect timeout=15)'))"))

moviepilot-moviepilot-1  | ERROR:   download.py - 万物生灵 (2020) 添加下载任务失败：All Creatures Great and Small S04 1080p Filmin WEB-DL AAC2.0 H.264-CHDWEB - https://ptchdbits.co/download.php?id=310248，添加种子任务失败：/config/temp/[CHDBits].All.Creatures.Great.and.Small.S04.1080p.Filmin.WEB-DL.AAC2.0.H.264-CHDWEB.torrent
moviepilot-moviepilot-1  | INFO:    chain - 发送消息：channel=None，title=添加下载任务失败：万物生灵 (2020) S04, text=站点：彩虹岛
moviepilot-moviepilot-1  | 种子名称：All Creatures Great and Small S04 1080p Filmin WEB-DL AAC2.0 H.264-CHDWEB
moviepilot-moviepilot-1  | 错误信息：添加种子任务失败：/config/temp/[CHDBits].All.Creatures.Great.and.Small.S04.1080p.Filmin.WEB-DL.AAC2.0.H.264-CHDWEB.torrent，userid=None

**Comment by Noir-Chen** [comment]
@kingcard1131 老哥，你这个问题解了么
